android/src/main/java/com/testfairy/flutter/TestfairyFlutterPlugin.java

```diff
@@ -711,6 +714,30 @@ private void disableAutoUpdate() {
 		TestFairy.disableAutoUpdate();
 	}
 
+	private void addUserInteraction(String kind, String label, final Map info) {
+		Map<String, String> sanitizedInfo = new HashMap<>();
+
+		if (info != null) {
+			if (info.containsKey("accessibilityLabel") && info.get("accessibilityLabel") != null) {
```

Comment:
Can you please replace this code with array?
String keys[] = ["accessbilityLabel", "accessibilityIdentifier", "accessibilityHint", "className"] and run through them?

Reply:
Done